### PR TITLE
Allow bundles without lang folders

### PR DESCRIPTION
**Pull request by hegrec**
### What
Allow themes that do not support translations and have no lang folder to be bundled successfully.

EAT-727

@mcampa @mickr

### CODE REVIEW

File: lib/langAssembler.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ module.exports.assemble = assemble;
 function assemble(callback) {
     var localeDirectory = 'lang';
 
+    try {
+        Fs.statSync(localeDirectory);
+    }
+    catch (e) {
+        // No lang folder, callback immediately
+        return callback(null, 'no_lang_folder');
```
Comment (mcampa) [inline lib/langAssembler.js]: I think you should return an error in the callback and then check the error type in `bin/stencil-bundle:294`

Comment (hegrec) [inline lib/langAssembler.js]: It's not really an error though, everything can still proceed, but it's there to provide an informational alert. An error should only be for something that keeps it from bundling.

Comment (mcampa) [inline lib/langAssembler.js]: I disagree with that, the return data, should be consistent, if the `results` variable is an object should always be an object. If the callback returns an error shouldn't indicate that the build has failed. You can set an error type or name and check for that name.

You could also check if the file exist before calling `assemble()`

Comment (hegrec) [inline lib/langAssembler.js]: Went forward with checking if it exists before calling assemble.

**Comment by mcampa** [comment]
:+1:

**Pull request merged**